libs/api/domains/syslumenn/src/lib/syslumenn.resolver.ts

```diff
@@ -137,6 +139,22 @@ export class SyslumennResolver {
     return this.syslumennService.getVehicleType(licenseNumber)
   }
 
+  @Query(() => VehicleRegistration)
+  syslumennGetVehicle(
+    @Args('input') input: GetVehicleInput,
+  ): Promise<VehicleRegistration> {
+    const response = this.syslumennService
+      .getVehicle(input.vehicleId)
+      .catch((e) => {
```

Comment:
![image](https://github.com/island-is/island.is/assets/43557895/fd93ad4e-e4e9-457b-90c2-385a221ba911)

Perhaps not having `e` might be better for lint purposes?

Reply:
Ah, yes

Reply:
I just removed the catch altogether since it is not needed